text/0000-sends-accepts.md

```diff
@@ -0,0 +1,46 @@
+- Start Date: 2015-12-3
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This proposes adding `sends` and `accepts` attributes to the `Category` element
```

Comment:
Swagger supports this data at the [root level](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#swagger-object) as well as the [operation level](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#operation-object).